pkg/virt-api/rest/subresource.go

```diff
@@ -908,48 +908,26 @@ func (app *SubresourceAPIApp) FilesystemList(request *restful.Request, response
 }
```

Comment:
Please give a consice commit description explained what is done in the commit and, if possible, try to shorten the title so that it fits in one line